libraries/helpers.rb

```diff
@@ -423,16 +416,6 @@ def efa_gdr_enabled?
   (config_value == enabling_value || config_value == "cluster") && graphic_instance?
 end
 
-# CentOS8 and alinux OSs currently not correctly supported by NFS cookbook
-# Overwriting templates for node['nfs']['config']['server_template'] used by NFS cookbook for these OSs
-# When running, NFS cookbook will use nfs.conf.erb templates provided in this cookbook to generate server_template
-def overwrite_nfs_template?
-  [
-    node['platform'] == 'amazon',
```

Comment:
Why deleting? I believe alinux and alinux2 both satisfy `node['platform'] == 'amazon'`

Reply:
Thanks, fixed!